prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/IL/il-phd.yml

```diff
@@ -0,0 +1,105 @@
+# To submit this to localhost, run:
```

Comment:
same comment as on ND. We shouldn't be storing the receiver settings here anymore as it is messing up platform's new feature to automatically check transforms.